### ISSUE TITLE
Data Size

### ISSUE THREAD

**Issue by Tarun712** [adamdavi3s/QlikviewSankeyImproved#3]
is there any limit on data that im using in this chart, as when im using this chart it goes in not responding mode.

**Comment by adamdavi3s** [comment]
Yes, obviously there is only so much the JS can handle! I think the original extension is better at handling more but I really need to review this
